election-guidelines-draft.md

```diff
@@ -0,0 +1,73 @@
+# ONNX Steering Commitee election guideline [DRAFT]
+
+## Introduction
+
+To encourage community participation and wider adoption in the industry, ONNX has introduced [open governance](https://github.com/onnx/onnx/wiki/Expanded-ONNX-Steering-Committee-Announced!) in March 2018. The governance has three defined structures to propel the development of ONNX project forward: [Steering Committee](https://github.com/onnx/onnx/tree/master/community#steering-committee), [Special Interest Groups (SIGs)](https://github.com/onnx/onnx/tree/master/community#sig---special-interest-groups), and [Working Groups (WGs)](https://github.com/onnx/onnx/tree/master/community#wg---working-groups). While SIGs and WGs primarily focus on the techinical roadmap of ONNX, the Steering Committe is responsible for setting the vision and governance process of the ONNX community. 
+
+For the first year of its ONNX open governance, representitives from Facebook, Microsoft, AWS, Intel and Nvidia are chosen to serve as the ONNX Steering Committee to help guide the project. The Steering Committee will be elected by the [Contributors](https://github.com/onnx/onnx/tree/master/community#community-roles) in its second year and will be re-elected every year. 
+
+This document is created to provide guidelines for the election process to ensure maximum transparency and fairness. 
+
+
+## Timeline
+
+Candidate applications will be accepted in April, and the election will be held in May. The new term for Steering Committee begins on June 1st of the corresponding year. The following table outlines the schedule for the election process. 
+
+| Schedule     | Event               | 
+|:-------------|:--------------------|
+| 1st Monday of April| Application for Steering Committee candidates open. |  
+| 3rd Monday of April| Candidates and their applications posted on github. Campaigning session held over VC.| 
+| 1st Monday of May| Election begins.     |
+| 2nd Monday of May| Election closes, and votes counted. Election results announced in the same week.|    
+| 3rd Monday of May| Previous Steering Committee to meet the newly elected Committee for official transition.| 
+| June 1 | New term begins with elected Steering Committee. Steering Committee Emeritus members help with the transition for the month of June.      |
+
+
+## Eligibility 
+
+### Eligibility for Steering Committee candadicy
+
+<code>**[TODO]** Candadicy eligbility needs to be discussed </code>
```

Comment:
*candidacy (sorry)

Reply:
Thanks for checking the typo! Will update :)

Reply:
Sorry, what is a campaigning session and who will be included in this VC?

Reply:
Campaign session would include candidates introducing themselves to voters (say 5 mins per candidate). Attendees will be candidates and voters.

Reply:
Got it. This doesnt seem scalable if we have global submissions and would like to have high attendance. Maybe we target to have candidates present at a workshop?